spec/edpm-openstack-service-architecture.md

```diff
@@ -0,0 +1,323 @@
+# OpenStack service deployment architecture on EDPM Node
+
+## Problem description
+
+Various OpenStack componentes have agents that need to run on the External
+DataPlane Node. For example OpenStack Nova's nova-api, nova-conductor
+and nova-scheduler services can run in kubernetes pods independently from the
+hypervisors they manage. In the other hand the nova-compute service is designed
+to be collocated with the hyperviors it manages. In the current design the EDPM
+Node serves as a hypervisor for Nova, therefore the nova-compute service needs
+to be deployed on EDPM nodes.
+
+Similar examples from other OpenStack compontents are ovn-metadata-agent, neutron-nicswitch agent, and in the HCI case the cinder-volume service.
```

Comment:
I think cinder-volume is in edge/DCN scenario on EDPM nodes, or has this changed? https://github.com/openstack/tripleo-heat-templates/blob/stable/train/roles/DistributedCompute.yaml#L28

Reply:
Sorry that is my mixup. Fixing it...